docs/troubleshooting.rst

```diff
@@ -193,3 +193,31 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+Can't override styles using Indigo Theme for MFEs
+-------------------------------------------------
+
+The indigo theme can’t override the styles for MFEs directly. It overrides styles for edx-platform. In case of MFEs, `@edx/brand <https://github.com/openedx/brand-openedx>`_ is used to override styles and then add your customized ``@edx/brand`` package in tutor-indigo. In this way, styles can be overidden::
```

Comment:
> In case of MFEs, `@edx/brand <https://github.com/openedx/brand-openedx>`_ is used to override styles and then add your customized ``@edx/brand`` package in tutor-indigo

Let's break this sentence into two.